src/storage/redis_metadata.h

```diff
@@ -202,7 +203,12 @@ class Metadata {
 
 class HashMetadata : public Metadata {
  public:
+  bool is_ttl_field_encoded;
```

Comment:
You can add a new enum type like:
```
enum class SubkeyEncoding {
  RAW = 0;
  WITH_TTL = 1;
};
```

Add then add a `SubkeyEncoding` field into `HashMetadata` instead of a `bool`.

Reply:
thank for you hint. I had updated it.